### ISSUE TITLE
Allow a max number of issues per violation

### ISSUE THREAD

**Issue by CaiqueCoelho** [CaiqueCoelho/robot-axe-library#4]
Why
Sometimes we want to allow a maximum number of issues per violation, for example, we should only get 2 buttons with a color contrast problem, if we get 3 we will return an error.

How
We should be able to pass a maximum number of issues allowed per violation in the keyword, and only return from the python code an error if the total of issues is greater than the  maximum allowed
